test-requirements.txt

```diff
@@ -1,7 +1,11 @@
-pytest_randomly~=3.8
-black==20.8b1
+black==22.1.0
+build
+isort
 pre-commit~=2.13
-wheel~=0.36
 pytest~=6.2
+pytest-cases~=3.6
 pytest-cov~=2.12
-pytest-cases~=3.6
\ No newline at end of file
+pytest_randomly~=3.8
+types-python-dateutil
+types-pytz
```

Comment:
Could you please add version numbers (either with >= or ~=) to the few python packages where none are defined